tests/run-travis.sh

```diff
@@ -21,6 +21,27 @@ function run_packages_tests {
 	done
 }
 
+function run_php_compatibility {
+	export PHPCOMP_EXEC="composer php:compatibility ."
+	export PHPCS_CHECK_EXEC="./vendor/bin/phpcs --version | grep -e PHP_CodeSniffer"
+	echo "Running PHP:Compatibility checks:"
+	echo "PHP Compatibility command: \`$PHPCOMP_EXEC\` "
+
+	if $PHPCS_CHECK_EXEC; then
+		# Everything is fine
+		:
+	else
+		exit 1
+	fi
+
+		if $PHPCOMP_EXEC; then
```

Comment:
indenting is now a bit off here.

```suggestion
	if $PHPCOMP_EXEC; then
```